modules/features/kids/src/main/java/au/com/shiftyjelly/pocketcasts/kids/KidsBottomSheetDialog.kt

```diff
@@ -28,7 +28,10 @@ import dagger.hilt.android.AndroidEntryPoint
 import javax.inject.Inject
 
 @AndroidEntryPoint
-class KidsBottomSheetDialog : BottomSheetDialogFragment() {
+class KidsBottomSheetDialog(
+    private val onFeedbackSentSuccess: () -> Unit,
+    private val onFeedbackSentError: () -> Unit,
+) : BottomSheetDialogFragment() {
```

Comment:
This will crash the app on configuration changes like the device rotation. Fragments must have an empty constructor.

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{au.com.shiftyjelly.pocketcasts.debug/au.com.shiftyjelly.pocketcasts.ui.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment au.com.shiftyjelly.pocketcasts.kids.KidsBottomSheetDialog: could not find Fragment constructor
```